tbot_contrib/gpio.py

```diff
@@ -0,0 +1,80 @@
+from tbot.machine import linux
+
+
+class Gpio:
+    def __init__(self, host: linux.LinuxShell, gpio_number: int):
+
+        self.host = host
+        self.gpio_number = gpio_number
+        self._gpio_sysclass_path = self.host.fsroot / "sys/class/gpio"
+        self._gpio_path = self._gpio_sysclass_path / f"gpio{self.gpio_number}"
+        self._export()
+        self._direction = self.get_direction()
+        self._active_low = self.get_active_low()
+
+    def _export(self) -> None:
+        if not self._gpio_path.is_dir():
+            (self._gpio_sysclass_path / "export").write_text(str(self.gpio_number))
+
+    def _is_in_direction(self) -> bool:
+        return self._direction == "in"
+
+    def _is_active_low_off(self) -> bool:
+        return not (self._active_low)
+
+    def set_direction(self, direction: str) -> None:
+        assert direction in ["in", "out"], f"Unsupported GPIO direction: {direction!r}"
+        if self._direction == direction:
+            return
+
+        (self._gpio_path / "direction").write_text(direction)
+        self._direction = direction
+
+    def set_active_low(self, value: bool) -> None:
+        if value:
+            (self._gpio_path / "active_low").write_text("1")
+            self._active_low = True
+        else:
+            (self._gpio_path / "active_low").write_text("0")
+            self._active_low = False
```

Comment:
This can be written much more concise like this:

```suggestion
        (self._gpio_path / "active_low").write_text("1" if value else "0")
        self._active_low = value
```

The same thing goes for a few functions further down, please change those as well.